content/lessons/chapter-2/hashing-1.tsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+
+import { useLocalizedRoutes, useTranslations } from 'hooks'
+import { ChapterIntro } from 'ui'
+import { Button } from 'shared'
+
+export const metadata = {
+  title: 'chapter_two.hashing_one.title',
+  key: 'CH2HSH1',
+}
+
+export default function Hashing1({ lang }) {
+  const routes = useLocalizedRoutes()
+  const t = useTranslations(lang)
+
+  return (
+    <ChapterIntro>
+      <p className="mt-2 text-lg md:text-xl">
+        {t('chapter_two.hashing_one.paragraph_one')}
+      </p>
+
+      <p className="mt-8 text-lg md:text-xl">
+        {t('chapter_two.hashing_one.paragraph_two')}
+      </p>
+
+      <p className="mt-8 text-lg md:text-xl">
+        {t('chapter_two.hashing_one.paragraph_three')}
+      </p>
+
+      <p className="mt-8 text-lg md:text-xl">
+        {t('chapter_two.hashing_one.paragraph_four')}
+      </p>
+
+      <p className="mt-8 text-lg md:text-xl">
+        {t('chapter_two.hashing_one.paragraph_five')}
+      </p>
+
+      <Button
+        href={`${routes.chaptersUrl}/chapter-2/hashing-2`}
```

Comment:
Instead of using a href here are we not able to use the new `useSaveAndProceed` hook I made? Using that would make it so that once a user presses this next button their progress is being stored, that doesn't seem to be the case now.